OpenTelemetry.sln

```diff
@@ -107,6 +107,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.Open
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.Prometheus.Tests", "test\OpenTelemetry.Exporter.Prometheus.Tests\OpenTelemetry.Exporter.Prometheus.Tests.csproj", "{393128EC-454D-45E8-A204-EC54904E5B3F}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Console", "Console", "{BC774012-9BDD-4742-876F-391E02EE6E67}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Web", "Web", "{2CE2987E-9791-43D4-8448-295543BEED7C}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.ApplicationInsights.Web", "samples\Exporters\Web\OpenTelemetry.Exporter.ApplicationInsights.Web.csproj", "{07989E94-E073-4E4F-8118-3927F458899D}"

```

Comment:
please add a link to example from README: https://github.com/open-telemetry/opentelemetry-dotnet#using-application-insights-exporter

Reply:
updated

Reply:
updated